pkgs/by-name/pp/ppfetch/package.nix

```
@@ -0,0 +1,29 @@
+{ stdenvNoCC, lib, fetchFromGitHub }:
+
+stdenvNoCC.mkDerivation rec {
```

Use of `finalAttrs` is generally preferred over `rec`. If this suggestion is accepted remember to close the newly opened parenthesis before `finalAttrs` at the end of the file.
```suggestion
stdenvNoCC.mkDerivation (finalAttrs: {
```